istioctl/pkg/waypoint/waypoint.go

```diff
@@ -61,6 +62,8 @@ const (
 	noTraffic       = "none"
 )
 
+var validAddressTypes = sets.New(serviceTraffic, workloadTraffic, allTraffic, noTraffic)
```

Comment:
Can we also swap the strings here with constant values like I do [here](https://github.com/istio/istio/blob/a0d6c0df39c8ce5207c2bc3740164b2841419f54/pkg/config/constants/constants.go#L202-L209)?